Functions.Templates/Documentation/tableIn.md

```diff
@@ -13,10 +13,9 @@
 
 The `table` binding can serialize or deserialize objects in JavaScript or C# functions. The objects will have RowKey and PartitionKey properties. 
 
-In C# functions, you can also bind to the following types:
+In C# functions, you can also bind to the following type:
 
-* `T` where `T` implements `ITableEntity`
-* `IQueryable<T>` 
+ * `T` where `T` implements `ITableEntity`

```

Comment:
There is also a CloudTable binding - you might as well mention that here as well